src/Core/API/addMovieAPI.js

```diff
@@ -0,0 +1,16 @@
+import api from "./axiosInstance";
+
+const addMovieAPI = async (payload) => {
```

Comment:
"async" keyword is redundant in the functions were there's not "await" used.
If some functions returns a promise, you can still add "await" when calling it inside "async" function.
`const addMoviesAPI = () => Promise.resolve(42);`
`const someFunction = async () => { ... await addMoviesAPI(...) ...};`